<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@839.v35e2736cfd5c">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@3565.v4b_d9b_8c29a_b_3">
    <parentIds>
      <string>21</string>
    </parentIds>
    <id>22</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.CoreStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@3565.v4b_d9b_8c29a_b_3">
      <arguments>
        <entry>
          <string>delegate</string>
          <org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable plugin="structs@324.va_f5d6774f3a_d">
            <symbol>appCenter</symbol>
            <arguments class="tree-map">
              <entry>
                <string>apiToken</string>
                <string>09dbebbcc47c6059225a410556bb0e6c7e82b37c</string>
              </entry>
              <entry>
                <string>appName</string>
                <string>myapp</string>
              </entry>
              <entry>
                <string>distributionGroups</string>
                <string>Beta Tester</string>
              </entry>
              <entry>
                <string>ownerName</string>
                <string>kapalkoko@gmail.com</string>
              </entry>
              <entry>
                <string>pathToApp</string>
                <string>build/app/outputs/apk/debug/app-debug.apk</string>
              </entry>
            </arguments>
          </org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable>
        </entry>
      </arguments>
      <sensitiveVariables/>
      <isUnmodifiedBySanitization>true</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@1200.v8005c684b_a_c6">
      <startTime>1672836860349</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@1200.v8005c684b_a_c6">
      <error class="java.util.concurrent.CompletionException">
        <detailMessage>io.jenkins.plugins.appcenter.AppCenterException: Create upload resource for app unsuccessful: HTTP 404 : {&quot;message&quot;:&quot;Not found. Correlation ID: d87e801f-e35e-459d-8e78-aadadf984a65&quot;,&quot;statusCode&quot;:404,&quot;code&quot;:&quot;Not Found&quot;}</detailMessage>
        <cause class="io.jenkins.plugins.appcenter.AppCenterException" plugin="appcenter@0.11.1">
          <detailMessage>Create upload resource for app unsuccessful: HTTP 404 : {&quot;message&quot;:&quot;Not found. Correlation ID: d87e801f-e35e-459d-8e78-aadadf984a65&quot;,&quot;statusCode&quot;:404,&quot;code&quot;:&quot;Not Found&quot;}</detailMessage>
          <stackTrace>
            <trace>io.jenkins.plugins.appcenter.AppCenterLogger.logFailure(AppCenterLogger.java:23)</trace>
            <trace>io.jenkins.plugins.appcenter.AppCenterLogger.logFailure(AppCenterLogger.java:37)</trace>
            <trace>io.jenkins.plugins.appcenter.task.internal.CreateUploadResourceTask.lambda$createUploadResourceForApp$0(CreateUploadResourceTask.java:58)</trace>
            <trace>java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859)</trace>
            <trace>java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837)</trace>
            <trace>java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)</trace>
            <trace>java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)</trace>
            <trace>retrofit2.CompletableFutureCallAdapterFactory$BodyCallAdapter$2.onResponse(CompletableFutureCallAdapterFactory.java:81)</trace>
            <trace>retrofit2.OkHttpCall$1.onResponse(OkHttpCall.java:129)</trace>
            <trace>okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)</trace>
            <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
            <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
            <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
          </stackTrace>
          <suppressedExceptions class="empty-list"/>
        </cause>
        <stackTrace>
          <trace>java.base/java.util.concurrent.CompletableFuture.encodeRelay(CompletableFuture.java:367)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture.completeRelay(CompletableFuture.java:376)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture$UniRelay.tryFire(CompletableFuture.java:1019)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)</trace>
          <trace>io.jenkins.plugins.appcenter.task.internal.CreateUploadResourceTask.lambda$createUploadResourceForApp$0(CreateUploadResourceTask.java:59)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)</trace>
          <trace>java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)</trace>
          <trace>retrofit2.CompletableFutureCallAdapterFactory$BodyCallAdapter$2.onResponse(CompletableFutureCallAdapterFactory.java:81)</trace>
          <trace>retrofit2.OkHttpCall$1.onResponse(OkHttpCall.java:129)</trace>
          <trace>okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>